lib/index_documents.rb

```diff
@@ -14,20 +14,28 @@ class IndexDocuments
   }
 
   def process(message)
-    if publishing_app_migrated?(message)
-      raw_links = links_from_payload(message)
-
-      unless raw_links.empty?
-        links = rummager_fields_from_links(raw_links)
-        base_path = document_base_path(message)
-        update_index(base_path, links)
-      end
-    end
-
+    $stdout.puts "Processing message: #{message.payload.inspect}"
+    index_links_from_message(message)
     message.ack
+  rescue GdsApi::HTTPServerError => e
+    $stderr.puts "An error occurred!"
+    $stderr.puts e.inspect
+    message.retry
```

Comment:
This has been added back? @rboulton told me to remove it and allow the message queue gem to handle this sort of thing.